lib/rubocop/cop/layout/empty_lines.rb

```diff
@@ -6,6 +6,20 @@ module RuboCop
   module Cop
     module Layout
       # This cops checks for two or more consecutive blank lines.
+      #
+      # @example
+      #
+      #   # bad - It has two empty lines.
+      #   some_method
+      #   # one empty line
+      #   # two empty lines
```

Comment:
I'm not sure about marking the empty lines this way, but YARD seems to remove empty lines from the docs.

Reply:
Seems reasonable.